### PR TITLE
Basket detail message for course entitlements

### PR DESCRIPTION
**Pull request by mjfrey**
Add custom message in the basket for course entitlements.

### CODE REVIEW

**Comment by mjfrey** [comment]
@edx/helio Take a look.

**Comment by catong** [comment]
@mjfrey FYI I'd like make suggestions that will include tweaks to the existing messages too - technically scope creep :-) but will make all messages more consistent.

File: ecommerce/extensions/basket/views.py [hunk 1]
```diff
@@ -234,16 +234,21 @@ def _process_basket_lines(self, lines):
...
-                        'You will be automatically enrolled in the verified track'
-                        ' of the course upon completing your order.'
-                    )
-                elif certificate_type == 'credit':
-                    order_details_msg = _('You will receive your credit upon completing your order.')
-                else:
+                if line.product.is_seat_product:
+                    if certificate_type == 'verified':
+                        order_details_msg = _(
+                            'You will be automatically enrolled in the verified track'
```
Comment (catong) [inline ecommerce/extensions/basket/views.py]: "After you complete your order you will be automatically enrolled in the verified track of the course."

**Comment by mjfrey** [comment]
@catong sure -- please recommend away.

File: ecommerce/extensions/basket/views.py [hunk 2]
```diff
@@ -234,16 +234,21 @@ def _process_basket_lines(self, lines):
...
+                            ' of the course upon completing your order.'
+                        )
+                    elif certificate_type == 'credit':
+                        order_details_msg = _('You will receive your credit upon completing your order.')
```
Comment (catong) [inline ecommerce/extensions/basket/views.py]: "After you complete your order you will receive credit for your course."
@mjfrey is "course" the correct context?

File: ecommerce/extensions/basket/views.py [hunk 3]
```diff
@@ -234,16 +234,21 @@ def _process_basket_lines(self, lines):
...
+                    else:
+                        order_details_msg = _(
+                            'You will be automatically enrolled in the course upon completing your order.'
```
Comment (catong) [inline ecommerce/extensions/basket/views.py]: "After you complete your order you will be automatically enrolled in the course."

File: ecommerce/extensions/basket/views.py [hunk 4]
```diff
@@ -234,16 +234,21 @@ def _process_basket_lines(self, lines):
...
+                        )
+                elif line.product.is_course_entitlement_product:
                     order_details_msg = _(
-                        'You will be automatically enrolled in the course upon completing your order.'
+                        'After completion of your order, you will be able to select course dates from your dashboard.'
```
Comment (catong) [inline ecommerce/extensions/basket/views.py]: "After you complete your order you will be able to select course dates from your dashboard."

**Review by catong — CHANGES_REQUESTED** [review]
Minor tweaks to make all order detail messages more consistent. I had one question about whether using "course" is correct for the "credit" cert type.

**Comment by mjfrey** [comment]
@catong changes made.

**Review by catong — APPROVED** [review]

**Comment by catong** [comment]
👍

**Pull request merged**